src/log-receiver/log-receiver.service.ts

```diff
@@ -17,68 +16,78 @@ import { Logs } from 'src/schema/logs.schema';
  */
 @Injectable()
 export class LogReceiverService {
+
   // Issue Creator for the Log Types
   cpuUtilizationIssueCreator: CpuUtilizationIssueCreatorComponent;
   timeoutIssueCreator: TimeoutIssueCreatorComponent;
   cbOpenIssueCreator: CbOpenIssueCreatorComponent;
   errorResponseIssueCreator: ErrorResponseIssueCreatorComponent;
-
   constructor(
     private http: HttpService,
     @InjectModel('logs') private logModel: Model<Logs>,
   ) {
     // Create an Issue Creator for each LogType
-    this.cpuUtilizationIssueCreator = new CpuUtilizationIssueCreatorComponent(
-      http,
-    );
+    this.cpuUtilizationIssueCreator = new CpuUtilizationIssueCreatorComponent(http);
     this.timeoutIssueCreator = new TimeoutIssueCreatorComponent(http);
     this.cbOpenIssueCreator = new CbOpenIssueCreatorComponent(http);
-    this.errorResponseIssueCreator = new ErrorResponseIssueCreatorComponent(
-      http,
-    );
+    this.errorResponseIssueCreator = new ErrorResponseIssueCreatorComponent(http);
   }
 
   /**
    * Handling of Log messages
-   *
    * @param logMessage is the log received by the log receiver controller
+   * @returns issueID that was received from the backend
    * This calls the handleLog of the corresponding IssueCreator and passed the log message
+   * 
```

Comment:
wrong order of lines. correct order is short description + blank line, more detailed description +  blank line (may be omitted if not necessary), tags.
```suggestion
   * 
   * @param logMessage is the log received by the log receiver controller
   * @returns issueID that was received from the backend
```